Commencement.Mvc/Web.config

```diff
@@ -24,6 +24,10 @@
     <add key="StaticCommencementUrl" value="http://commencement.ucdavis.edu/registration.html" />
     <add key="ReportToken" value="EXTERNAL" />
     <add key="ReportApiUrl" value="https://test.caes.ucdavis.edu/ReportService/api/honors" />
+    <add key="Stackify.AppName" value="Commencement" />

```

Comment:
@jSylvestre Each of the jobs should also have an AppSettings.config file with this line in it, setting the appname to something like Commecement.Jobs.JobName.